### PR TITLE
[docs] - supercollider/HelpSource/Classes/MIDIOut.schelp - reviewing grammar

### PR DESCRIPTION
**Pull request by tem44**
#### [docs] - supercollider/HelpSource/Classes/MIDIOut.schelp - reviewing grammar

### CODE REVIEW

File: HelpSource/Classes/MIDIOut.schelp [hunk 1]
```diff
@@ -14,7 +14,7 @@ private::connectByUID, disconnectByUID
 
 method::new
 
-Create a new MIDIOut instance. Note that this method is not safe for cross-platform usage with Linux, because the meaning of the teletype::port:: argument is different. See link::#Linux specific: Connecting and disconnecting ports:: for details.
+Create a new MIDIOut instance. Note that this method is not safe for cross-platform usage with Linux, because the meaning of the teletype::port:: the argument is different. See link::#Linux specific: Connecting and disconnecting ports:: for details.
```
Comment (snappizz) [inline HelpSource/Classes/MIDIOut.schelp]: the addition of this "the" is not correct.

File: HelpSource/Classes/MIDIOut.schelp [hunk 2]
```diff
@@ -123,11 +123,11 @@ MIDIClient.init;
 
 teletype::
 MIDI Sources:
-	MIDIEndPoint("System", "Timer")
-	MIDIEndPoint("System", "Announce")
-	MIDIEndPoint("Midi Through", "Midi Through Port-0")
-	MIDIEndPoint("SuperCollider", "out0")
-	MIDIEndPoint("SuperCollider", "out1")
+    MIDIEndPoint("System", "Timer")
```
Comment (snappizz) [inline HelpSource/Classes/MIDIOut.schelp]: please do not change indentation here

**Comment by tem44** [comment]
@snappizz can you double check and provide some feedback on those. thank you so much

File: HelpSource/Classes/MIDIOut.schelp (hunk 1 above)
Comment (snappizz) [reply HelpSource/Classes/MIDIOut.schelp]: the word "the" is still here.

File: HelpSource/Classes/MIDIOut.schelp [hunk 3]
```diff
@@ -14,7 +14,7 @@ private::connectByUID, disconnectByUID
 
 method::new
 
-Create a new MIDIOut instance. Note that this method is not safe for cross-platform usage with Linux, because the meaning of the teletype::port:: argument is different. See link::#Linux specific: Connecting and disconnecting ports:: for details.
+Create a new MIDIOut instance. Note that this method is not safe for cross-platform usage with Linux, because the meaning of teletype::port:: the argument is different. See link::#Linux specific: Connecting and disconnecting ports:: for details.
```
Comment (snappizz) [inline HelpSource/Classes/MIDIOut.schelp]: this is still incorrect.

**Comment by tem44** [comment]
hey @snappizz  please let me know if there's still something needing to be fixed. thank you so much

**Pull request merged**